### DISCUSSION TITLE
Multiple record options per camera

### DISCUSSION THREAD

**Discussion by BurningTheSky** [blakeblackshear/frigate#7612] [Ask A Question]
I'm just getting set up with Frigate for the first time, moving over from Zoneminder, and I have a 1050TI which I plan to use for de/encoding and dection until I fully commit and get a coral. I'm looking at recreating part of a config I have in ZM where I do 24/7 recording on the sub feed which is at 640x360 to reduce space used, then record motion on main feed at 1440p. I do this by having 2 cameras configured. Do I need to to the same with Frigate or is there another way to do it?

Any other tips or suggestions would be greatly received as well!

**Comment by NickM-27** [comment]
There's no official way to do this currently other than 2 separate cameras

**Comment by BurningTheSky** [comment]
Thank you, I'll proceed on that basis
